runtime/queries/tact/highlights.scm

```diff
@@ -0,0 +1,210 @@
+; NOTE: Order of highlight queries matters, as Tree-sitter uses the first it finds.
+; NOTE: Therefore, narrow highlight queries should be placed before broad captures.
+; ---------------------------------------------------------------------------------
+
+; attribute
+; ---------
+
+[
+  "@name"
+  "@interface"
+] @attribute
+
+; comment
+; -------
+
+(comment) @comment
+
+; function.builtin
+; ----------------
+
+((identifier) @function.builtin
+  (#match? @function.builtin
+     "^(send|sender|require|now|myBalance|myAddress|newAddress|contractAddress|contractAddressExt|emit|cell|ton|dump|beginString|beginComment|beginTailString|beginStringFromBuilder|beginCell|emptyCell|randomInt|random|checkSignature|checkDataSignature|sha256|min|max|abs|pow|throw|nativeThrowWhen|nativeThrowUnless|getConfigParam|nativeRandomize|nativeRandomizeLt|nativePrepareRandom|nativeRandom|nativeRandomInterval|nativeReserve)$")
```

Comment:
Since #9317 you can use the `#any-of?` predicate instead of these `#match?`es:

```scm
((identifier) @function.builtin
 (#any-of? @function.builtin "send" "sender" "require" ; ... etc. ...
))
```